docs/src/pages/components/authenticator/formfields/qrcode/form-fields.react.mdx

```diff
@@ -0,0 +1,19 @@
+```tsx{3-9, 17}
+
+const formFields = {
+  setupTOTP: {
+        QR: {
+          totpIssuer: 'test issuer',
+          totpUsername: 'amlify_qr_test_user',
```

Comment:
I think you have a typo here... amlify_qr_test_user vs amplify_qr_test_user

Reply:
Thanks for the catch! #1452 addresses it.